netadapter/server/grpcserver/grpc_server.go

```diff
@@ -37,6 +37,10 @@ func NewGRPCServer(listeningAddrs []string) (server.Server, error) {
 }
 
 func (s *gRPCServer) Start() error {
+	if s.onConnectedHandler == nil {
```

Comment:
We agreed that only grpcConnection is allowed to panic immidiately, grpcServer can return error up to app.Start, which will panic for it.